public/app/plugins/datasource/tempo/metricsSummary.ts

```diff
@@ -183,7 +183,7 @@ const getConfig = (series: Series, query: string, instanceSettings: DataSourceIn
   return { ...commonConfig };
 };
 
-const NO_VALUE = '';
+const NO_VALUE = '<no value>';
```

Comment:
Maybe it would make sense instead to use a field config with mapping. That way the value will be still considered as empty string for sorting but will show `<no value>`. I guess it's a slight chance but now this can sort differently if there are actual values with `<` at the start or other special chars.

Reply:
Actually users may want to sort by `<no value>`. I don't think introducing sorting behaviour is necessary right now and would like to merge this as is.